### ISSUE TITLE
Make configuration list interactive

### ISSUE THREAD

**Issue by jabberwock** [synfinatic/aws-sso-cli#800]
**Is your feature request related to a problem? Please describe.**
I would like to be able to pull up `aws-sso list` but be able to, with a keyboard/mouse/tap/etc., select the profile to use.

**Describe the solution you'd like**
I made this PoC for bash/zsh, and it requires the `fzf` program to be installed:

```bash
function aws-sso-select() {
    local aws_profile=$(aws-sso list | tail -n +5 | fzf --bind 'enter:execute(echo {7})+abort')
    eval $(/usr/local/bin/aws-sso eval -p $aws_profile)
}
```

**Describe alternatives you've considered**
The alternative is shown in the PoC above. It works but I'd like to see it natively supported.

**Additional context**
This simply saves me some time having to type `aws-sso-profile` and pasting the profile. That's all. :)

**Comment by jabberwock** [comment]
No no. I was thinking of implementing it in Go. That fzf proof-of-concept is just that. I noticed you already have some  keyboard/mouse interaction during the initial setup for selecting things and I figured it wouldn't be "too" hard to implement, but that could be my ignorance.  I'll have a look at the codebase tomorrow and see if it's something I can do. Thanks! 

-jw

**Comment by synfinatic** [comment]
Thinking about this, this is basically using `ProfileFormat` with the search function

**Comment by synfinatic** [comment]
I'm unlikely to add support for this unless someone does the heavy lifting... managing and supporting all the various shells is a PITA.  Will see about adding interactive mode selection option for Profile's though.

**Comment by jabberwock** [comment]
Agreed. Something this "GUI" may be better off having a cross-platform GUI frontend than trying to make all shells happy.
